src/MentionsInput.js

```diff
@@ -234,8 +234,9 @@ class MentionsInput extends React.Component {
 
   // Handle input element's change event
   handleChange = (ev) => {
-
-    if(document.activeElement !== ev.target) {
+    // if we are inside iframe, we need to find activeElement within its contentDocument
+    let currentDocument = document.activeElement.contentDocument || document;
```

Comment:
Can `const` be used here?

Also, is it possible to have a scenario where `document.activeElement` is null?

Reply:
1. Seems overall style of this library is to use let, not sure why
2. Nope, it doesn't seem to be possible, some element is always active (body by default)

Reply:
👍

Reply:
I'd prefer using `const` here. We should improve usage of `const` vs. `let` this in the rest of the code (among many other things), too.

Reply:
We might want to guard against `document.activeElement` being `undefined`. Not sure if this can really happen, but the comment in the line below `// fix an IE bug (blur from empty input element with placeholder attribute trigger "input" event)` kinda points in that direction...

Reply:
Yeah, agreed.